spec/dummy/app/views/layouts/application.html.erb

```diff
@@ -5,9 +5,10 @@
     <%= csrf_meta_tags %>
   </head>
   <body>
-
     <%= yield %>
 
-    <%= include_access_lint %>
+    <% if Rails.env.test? || Rails.env.development? %>
```

Comment:
Being this the specs dummy app, is there a reason why we would need the conditional?